driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedCrudHelper.java

```diff
@@ -379,9 +379,9 @@ OperationResult createFindCursor(final BsonDocument operation) {
     @NonNull
     private FindIterable<BsonDocument> createFindIterable(final BsonDocument operation) {
         MongoCollection<BsonDocument> collection = entities.getCollection(operation.getString("object").getValue());
-        BsonDocument arguments = operation.getDocument("arguments");
+        BsonDocument arguments = operation.getDocument("arguments", new BsonDocument());
         ClientSession session = getSession(arguments);
-        BsonDocument filter = arguments.getDocument("filter", new BsonDocument());
```

Comment:
This matches all the other places where `filter` is required.  I had to make this change after changing the line two lines up, as the exception thrown on that line when there are no arguments was incorrectly causing the `createFindCursor fails if filter is not specified` in `unified-test-format/valid-fail` to pass.